_pages/icml_2020/icml-2020-presenters.html

```diff
@@ -1,10 +1,52 @@
 ---
-layout: main
+layout: main.icml
 permalink: /icml-2020-presenters
 title: LXAI @ ICML - Presenters
-published: false
+published: true
 
 ---
 <div id='icmlpresenters2020'>
+    <div class="workshop-header" id="workshopHeader">
```

Comment:
@kevinsbello the nav should be brought in as an includes file see prior workshop page: https://github.com/latinxinai/latinxinai.github.io/blob/master/_pages/neurips_2018/neurips-2018-presenters.html